frontend/components/maps/MapTest.js

```diff
@@ -0,0 +1,269 @@
+                religionMarkers.push(
+                    <Marker key={i.toString()} position = {coords}>
+                        <Popup>
+                            <p>{geos[i]["name"]}</p>
+                        </Popup>
+                    </Marker>);
+            }
+
+        }
+
+        // eslint-disable-next-line max-len
+        const c = ["#56ebd3", "#115e41", "#88beef", "#4346ab", "#b88ae6", "#8711ac", "#e84fe1", "#1d70a5", "#37c454", "#c2df7d", "#719d47", "#1fa198", "#f8cca6", "#76480d", "#e37010", "#871d32", "#f48e9b", "#e21c7a", "#f73931", "#f9bd3a"];
+        const redOptions = {color: "red"};
+
+        //console.log("here ", this.props.voronoi);
+        let shapes = [];
+        let v = this.props.voronoi;
+        for (let i = 0; i < v.length; i++) {
+            if (v[i]["year"] === this.props.decade) {
+                for (let j=0; j< v[i]["shapes"].length; j++) {
+                    shapes.push(
+                        // eslint-disable-next-line max-len
+                        <Polygon id={j.toString()} pathOptions={{color:c[j]}} positions={v[i]["shapes"][j]}/>
+                    );
+                }
+            }
+        }
+        let omie = [];
+        omie.push(
+            // eslint-disable-next-line max-len
+            <Circle id={100} center={[38.89874393140099, -76.92357341077269]} pathOptions={redOptions} radius={75}/>
+        );
+
+        let paths = this.props.paths;
+        console.log(paths);
+
+        for (let i = 0; i < paths.length; i++) {
+            if (paths[i]["decade"] === this.props.decade) {
+                for (let j=0; j< paths[i]["paths"].length; j++) {
+                    console.log("path ", paths[i]["paths"][j]["path"]);
+                    omie.push(
+                        // eslint-disable-next-line max-len
+                        <Polyline id={j.toString()} pathOptions={redOptions} positions={paths[i]["paths"][j]["path"]} pane={"markerPane"}>
+                            {/* eslint-disable-next-line max-len */}
+                            <Tooltip sticky>{paths[i]["paths"][j]["distance"]} miles in {paths[i]["paths"][j]["time"]} minutes</Tooltip>
+                        </Polyline>
+                    );
+                }
+            }
+        }
+
+
+
+
+
+        let L = [];
+        //console.log(this.props.mapType);
+        if (this.props.mapType === "Food") {
+            L = [
+
+
+                <LayersControl.Overlay checked key={1} name="Voronoi Representation">
+                    <LayerGroup>{shapes}</LayerGroup>
+                </LayersControl.Overlay>,
+
+                <LayersControl.Overlay key={2} name="Census Tract Boundaries">
+                    <LayerGroup>{Object.keys(this.state.censustract).length > 0 &&
+                    <GeoJSON data={this.state.censustract}/>}</LayerGroup>
+                </LayersControl.Overlay>,
+
+                <LayersControl.Overlay key={2} name="Deanwood Boundaries">
```

Comment:
keys should be unique! You have two layer 2s here. Since we're hand-coding them anyway, I'm adding "layer-" to each num